### PR TITLE
Feature/civ 923 leo

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/dashboard/claimSummaryController.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const {Logger} = require('@hmcts/nodejs-logging');
+const logger = Logger.getLogger('claimSummaryController');
+
+function getComponentDetailItems(lang: string): ComponentDetailItems {
+  return {
+    title: t('PAGES.CHECK_YOUR_ANSWER.DETAILS_TITLE', {lng: getLng(lang)}),
+    subtitle: t('PAGES.CHECK_YOUR_ANSWER.DETAILS_TITLE', {lng: getLng(lang)}),
+    content: [
+      '<p class="govuk-body">If you’ve been paid or you’ve made another agreement with the defendant, you need to tell us.</p>',
+      '<p class="govuk-body">More information about the <a class="govuk-link" href="https://www.gov.uk/government/publications/debt-respite-scheme-breathing-space-guidance/debt-respite-scheme-breathing-space-guidance-for-creditors#end-of-a-breathing-space">debt respite scheme (opens in new tab)</a></p>',
```
Comment (laarmada) [inline src/main/routes/features/dashboard/claimSummaryController.ts:21]: where is t function to translate

File: src/main/routes/features/dashboard/claimSummaryController.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const claimSummaryController = express.Router();
+const {Logger} = require('@hmcts/nodejs-logging');
+const logger = Logger.getLogger('claimSummaryController');
+
+function getComponentDetailItems(lang: string): ComponentDetailItems {
+  return {
+    title: t('PAGES.CHECK_YOUR_ANSWER.DETAILS_TITLE', {lng: getLng(lang)}),
+    subtitle: t('PAGES.CHECK_YOUR_ANSWER.DETAILS_TITLE', {lng: getLng(lang)}),
+    content: [
+      '<p class="govuk-body">If you’ve been paid or you’ve made another agreement with the defendant, you need to tell us.</p>',
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/routes/features/dashboard/claimSummaryController.ts:20]: or key?

**Review by laarmada — CHANGES_REQUESTED** [review]
hard job!

File: src/main/routes/features/dashboard/claimSummaryController.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (GorilaNaranja) [reply src/main/routes/features/dashboard/claimSummaryController.ts:20]: This component is only a probe of concept to check if we can pass html as parameter.

**Review by laarmada — APPROVED** [review]
ok

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -234,6 +234,16 @@
       "TITLE": "Why do you disagree with the claim?",
       "REASON": "Their reasons for making the claim",
       "CLAIMANT": "Your response will be sent to {{ claimantFullName }}."
+    },
+    "CLAIM_DETAILS": {
```
[the remark this answers is not in the document]
Comment (GorilaNaranja) [reply src/main/modules/i18n/locales/cy.json]: Updated

**Pull request merged**